waku/v2/protocol/rln/waku_rln_relay.go

```diff
@@ -131,30 +132,30 @@ func (rlnRelay *WakuRLNRelay) ValidateMessage(msg *pb.WakuMessage, optionalTime
 	if int64(math.Abs(float64(gap))) > maxEpochGap {
 		// message's epoch is too old or too ahead
 		// accept messages whose epoch is within +-MAX_EPOCH_GAP from the current epoch
-		rlnRelay.log.Debug("invalid message: epoch gap exceeds a threshold", zap.Int64("gap", gap))
+		rlnRelay.log.Info("invalid message: epoch gap exceeds a threshold", zap.Int64("gap", gap))
 		rlnRelay.metrics.RecordInvalidMessage(invalidEpoch)
 
 		return invalidMessage, nil
 	}
 
 	if !(rlnRelay.RootTracker.ContainsRoot(msgProof.MerkleRoot)) {
-		rlnRelay.log.Debug("invalid message: unexpected root", logging.HexBytes("msgRoot", msgProof.MerkleRoot[:]))
+		rlnRelay.log.Info("invalid message: unexpected root", logging.HexBytes("msgRoot", msgProof.MerkleRoot[:]))
 		rlnRelay.metrics.RecordInvalidMessage(invalidRoot)
 		return invalidMessage, nil
 	}
 
 	start := time.Now()
 	valid, err := rlnRelay.verifyProof(msg, msgProof)
 	if err != nil {
-		rlnRelay.log.Debug("could not verify proof", zap.Error(err))
+		rlnRelay.log.Info("could not verify proof", zap.Error(err))
 		rlnRelay.metrics.RecordError(proofVerificationErr)
 		return invalidMessage, nil
 	}
```

Comment:
This should probably be debug or error. Also something interesting i noticed thanks to this change - since this is a technical error, probably makes sense to return a `validationError`!

Reply:
Fixed at https://github.com/waku-org/go-waku/pull/1003/commits/924d5a5b184f73f28ac9118fe391a8bef0ee0a18